cypress/e2e/basic-starter.cy.ts

```diff
@@ -160,7 +160,7 @@ describe('Node page', () => {
 
   it('should render an event node', () => {
     cy.visit(
-      '/event/webinar/2022/08/event-two-medium-length-placeholder-heading',
+      '/event/webinar/2022/09/event-two-medium-length-placeholder-heading',
```

Comment:
This one appears to have changed out from under us. Not sure why, but it's an easy adjustment.

Reply:
@phenaproxima I think you should use static node id for this or else you've to add logic something like `current_month + 1`. The reason is, the alias is dynamic as it depends on future event date. See this: [ACMS-761](https://backlog.acquia.com/browse/ACMS-761)

Reply:
Thanks for the helpful context @vishalkhode1! 💯 

I'm wondering if we could add a step in our container build that would make this path consistent? I don't think we can do `current_month +1` in the test since we might not update the container exactly when the month changes.

Reply:
@vishalkhode1, I'll try to change this in our container build. We _definitely_ cannot rely on node IDs (or any kind of serial ID) for anything, because they are not static. They are likely to change every time we rebuild the container (i.e., every time the starter kit is reinstalled).